aif360/aif360-r/R/import.R

```diff
@@ -3,16 +3,21 @@
 #'
 load_aif360_lib <- function() {
   e <- globalenv()
-  e$datasets <- import("aif360.datasets")
-  e$metrics <- import("aif360.metrics")
-  e$pre_algo <- import("aif360.algorithms.preprocessing")
-  e$in_algo <- import("aif360.algorithms.inprocessing")
-  e$post_algo <- import("aif360.algorithms.postprocessing")
-  e$tf <- import("tensorflow")
-  lockBinding("datasets", e)
-  lockBinding("metrics", e)
-  lockBinding("pre_algo", e)
-  lockBinding("in_algo", e)
-  lockBinding("post_algo", e)
-  lockBinding("tf", e)
+  bindings <- c("datasets", "metrics", "pre_algo", "in_algo", "post_algo", "tf")
+  if (!all(bindings %in% ls(e))){
+    e$datasets <- import("aif360.datasets")
+    e$metrics <- import("aif360.metrics")
+    e$pre_algo <- import("aif360.algorithms.preprocessing")
+    e$in_algo <- import("aif360.algorithms.inprocessing")
+    e$post_algo <- import("aif360.algorithms.postprocessing")
+    e$tf <- import("tensorflow")
+    lockBinding("datasets", e)
+    lockBinding("metrics", e)
+    lockBinding("pre_algo", e)
+    lockBinding("in_algo", e)
+    lockBinding("post_algo", e)
+    lockBinding("tf", e)
+  } else {
+    message(paste0(c("The aif360 functions have already been loaded:", bindings), sep = " "))
```

Comment:
Two suggestions.

- Let's not display all the functions as it is not required for the users
- Expand the message to something like `The aif360 functions have already been loaded. You can begin using the package`